db/comdb2.c

```diff
@@ -5715,8 +5715,7 @@ static struct dbenv *newdbenv(char *dbname, char *lrlname)
         return NULL;
     }
 
-    if (lrlname)
-        pre_read_lrl_file(dbenv, lrlname, dbname);
+    if (lrlname) pre_read_lrl_file(dbenv, lrlname, dbname);
```

Comment:
Well, you can try with AllowShortIfStatementsOnASingleLine: false, or try omitting it totaly.
I did not think we decided to force if stmts on same line -- if we did that's silly and we should not enforce.